src/src/AppBundle/Controller/DefaultController.php

```diff
@@ -3,17 +3,20 @@
 namespace AppBundle\Controller;
 
 use AdminBundle\Entity\EventType;
+use AppBundle\Entity\Attendee;
+use AppBundle\Form\RegistrationForm;
+use EmailBundle\Controller\EmailController;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use TemplateBundle\Controller\CustomTemplateController;
+use Symfony\Component\HttpFoundation\Request;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 
-class DefaultController extends CustomTemplateController
+class DefaultController extends EmailController
```

Comment:
Bolo by treba ohandlovat ked sa user snazi dostat na neexistujuci slug alebo cisto len na "/". Malo by sa to ale najskor riesit v inom issue.

Reply:
No momentalne to hodi 404ku, co je to spravne riesenie. Ano, ukaze to hnusny error message ale  to kvoli tomu, ze sme v debug mode, na prdokucii by to malo vytiahnut zo symfony nejaku (peknu?) 404ku.

Reply:
Ok, ak staci 404 tak v pohode